app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ReviewsController < ApplicationController
+
+  def new
+    # @shelter = Shelter.find(params[:id])
+    @shelter_id = params[:shelter_id]
```

Comment:
this :shelter_id needs to match with the route. Where it used to be shelters/:id it is shelters/:shelter_id whenever it relates to reviews